build.sh

```diff
@@ -62,11 +70,11 @@ PROPRIETARY_FIRMWARE_OUTPUT_DIR="${DEBIAN_DIR}/lib/firmware"
 PROPRIETARY_FIRMWARE_INITRAMFS_DIR="${INITRAMFS_DST_DIR}/lib/firmware"
 
 
-BB_VERSION="1.31.0"
+BB_VERSION="1.36.1"
```

Comment:
Perhaps we can store this version on cloudsmith so that we always have it - until we need to upgrade again. Is this alwasy being downloaded and unzipped everytime a fresh build is performed?